includes/MediaWiki/Api/EditEndpoint.php

```diff
@@ -60,7 +60,7 @@ public function run() {
 
 		if ( $this->requestBody === null ) {
 			throw new LocalizedHttpException(
-				MessageValue::new( 'wikibasereconcileedit-editendpoint-invalid-request-body' ),
+				MessageValue::new( 'wikibasereconcileedit-invalid-JSON-request-body' ),
```

Comment:
this JSON needs to be lowercase now

Reply:
:scream: omg true.. this is what you get from trying to fix things during meetings :D i hope it's ok now